src/views/v2/commerce/checkout/cart/billing_identification.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Tickets Commerce: Billing Identification Fields
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/cart/billing_identification.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ * @var bool $must_login [Global] Whether login is required to buy tickets or not.
+ */
+
+if ( $must_login ) {
+	return;
+}
+
+$user      = wp_get_current_user();
+$username  = '';
+$useremail = '';
+
+if ( ! empty( $user ) ) {
+	$username  = $user->display_name;
+	$useremail = $user->user_email;
+}
+
+?>
+
+<div id="tec-tc-gateway-stripe-billing-identification">
```

Comment:
Maybe we need to create a follow-up ticket to be sure that this is following the FE standards. @codingmusician I'll ping you with a ticket to take care of these templates.